src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -0,0 +1,283 @@
+        BEAST_EXPECT(jrr[jss::ledger_hash] == to_string(env.closed()->info().hash));
+        BEAST_EXPECT(jrr[jss::ledger_index] == 3);
+
+        Json::Value jvParams;
+        jvParams[jss::account_root] = alice.human();
+        jvParams[jss::ledger_hash] = jrr[jss::ledger_hash];
+        jrr = env.rpc ( "json", "ledger_entry", to_string(jvParams) ) [jss::result];
+        BEAST_EXPECT(jrr.isMember(jss::node));
+        BEAST_EXPECT(jrr[jss::node][jss::Account] == alice.human());
+        BEAST_EXPECT(jrr[jss::node][sfBalance.fieldName] == XRP(10000).value().getText());
+    }
+
+    void testMalformedAccountRoot()
+    {
+        testcase("Malformed Ledger Entry Request");
+        using namespace test::jtx;
+        Env env {*this};
+        Account const alice {"alice"};
+        env.fund(XRP(10000), alice);
+        env.close();
+
+        auto jrr = env.rpc("ledger_closed") [jss::result];
+
+        Json::Value jvParams;
+        jvParams[jss::account_root] = std::string(alice.human()).replace(0, 2, 2, 'x');
+        jvParams[jss::ledger_hash] = jrr[jss::ledger_hash];
+        jrr = env.rpc ( "json", "ledger_entry", to_string(jvParams) ) [jss::result];
+        BEAST_EXPECT(jrr[jss::error] == "malformedAddress");
+        BEAST_EXPECT(jrr[jss::status] == "error");
+        BEAST_EXPECT(jrr[jss::error_message] == Json::nullValue);
+    }
+
+    void testNotFoundAccountRoot()
+    {
+        testcase("Ledger Entry Not Found");
+        using namespace test::jtx;
+        Env env {*this};
+        Account const alice {"alice"};
+        env.fund(XRP(10000), alice);
+        env.close();
+
+        auto jrr = env.rpc("ledger_closed") [jss::result];
+
+        Json::Value jvParams;
+        jvParams[jss::account_root] = Account("bob").human();
+        jvParams[jss::ledger_hash] = jrr[jss::ledger_hash];
+        jrr = env.rpc ( "json", "ledger_entry", to_string(jvParams) ) [jss::result];
+        BEAST_EXPECT(jrr[jss::error] == "entryNotFound");
+        BEAST_EXPECT(jrr[jss::status] == "error");
+        BEAST_EXPECT(jrr[jss::error_message] == Json::nullValue);
+    }
+
+    void testAccountRootFromIndex()
+    {
+        testcase("Ledger Entry Request From Index");
+        using namespace test::jtx;
+        Env env {*this};
+        Account const alice {"alice"};
+        env.fund(XRP(10000), alice);
+        env.close();
+
+        auto jrr = env.rpc("ledger_closed") [jss::result];
+        BEAST_EXPECT(jrr[jss::ledger_hash] == to_string(env.closed()->info().hash));
+        BEAST_EXPECT(jrr[jss::ledger_index] == 3);
+
+        Json::Value jvParams;
+        jvParams[jss::account_root] = alice.human();
+        jvParams[jss::ledger_hash] = jrr[jss::ledger_hash];
+        jrr = env.rpc ( "json", "ledger_entry", to_string(jvParams) ) [jss::result];
+        jvParams[jss::index] = jrr[jss::index];
+        jrr = env.rpc ( "json", "ledger_entry", to_string(jvParams) ) [jss::result];
+        BEAST_EXPECT(jrr.isMember(jss::node_binary));
```

Comment:
In addition to checking that it exists, check the value:
```
        BEAST_EXPECT(jrr[jss::node_binary] ==
            "1100612200800000240000000225000000032D00000000554294BEBE5B56"
            "9A18C0A2702387C9B1E7146DC3A5850C1E87204951C6FDAA4C4262400000"
            "02540BE4008114AE123A8556F3CF91154711376AFB0F894F832B3D");
```
AFAIK, this stuff is deterministic, so this value shouldn't change. If somebody breaks something and it does, though, yahtzee!